ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -265,24 +293,33 @@ if [ $verify = upper ] ; then
         export err=$?; err_chk
       fi
     fi
-  done # end of metplus_jobs
+  done # end of metplus_jobs loop
+
+  # Indicate all tasks are completed
+  >$COMOUTsmall/stats_completed
+  echo "stats are completed" >> $COMOUTsmall/stats_completed
+
+  # Clean up restart directory
+  rm -rf $COMOUTsmall/restart/grid2grid/run_${modnam}_valid_at_*
```

Comment:
I don't think we can be removing directories from COMOUTsmall like this.

@AliciaBentley-NOAA @AndrewBenjamin-NOAA @PerryShafran-NOAA Thoughts?

Reply:
I agree with Mallory.  There is no need to remove this directory; most other restart directories are kept online.  And I would guess that NCO would balk at it anyway.   Plus restart directories need to be kept online in case of restart.

Reply:
Good to know. I will delete this line of code.